client/src/pages/SubmissionView.vue

```diff
@@ -0,0 +1,507 @@
+<template>
+  <div v-if="!submission" class="q-pa-lg">
+    {{ $t("loading") }}
+  </div>
+  <article v-else>
+    <q-layout
+      data-cy="submission_view_layout"
+      view="hHh lpR fFr"
+      container
+      style="min-height: calc(100vh - 70px)"
+    >
+      <q-header reveal elevated class="bg-grey-9 text-white">
+        <q-toolbar>
+          <q-btn
+            dense
+            aria-label="Back to Submission Details"
+            flat
+            round
+            icon="arrow_back_ios_new"
+            :to="{
+              name: 'submission_details',
+              params: { id: props.id },
+            }"
+          />
+          <q-toolbar-title>
+            {{ submission.title }}
+          </q-toolbar-title>
+
+          <q-btn
+            aria-label="Toggle Inline Comments"
+            dense
+            flat
+            round
+            icon="question_answer"
+            @click="toggleRightDrawer"
+          />
+        </q-toolbar>
+      </q-header>
+
+      <q-drawer
+        v-model="rightDrawerOpen"
+        show-if-above
+        side="right"
+        bordered
+        :width="drawerWidth"
+      >
+        <div class="row fit">
+          <div
+            v-touch-pan.horizontal.prevent.mouse.preserveCursor="handlePan"
+            style="width: 6px; cursor: col-resize"
+            class="bg-primary"
+          ></div>
+          <q-scroll-area class="fit col bg-grey-4">
+            <inline-comment :user="currentUser" />
+            <div class="q-ml-md q-mb-md">
+              <inline-comment-reply :user="currentUser" />
+              <inline-comment-reply :user="currentUser" />
+            </div>
+            <div class="row justify-center q-pa-md">
+              <q-btn color="dark" icon="arrow_upward">Scroll to Top</q-btn>
+            </div>
+          </q-scroll-area>
+        </div>
+      </q-drawer>
```

Comment:
I'd also make this a component, SubmissionReviewInlineDrawer

Reply:
This has been refactored as suggested.